tests/baselines/reference/contextualTypingWithFixedTypeParameters1.errors.txt

```diff
@@ -10,8 +9,7 @@ tests/cases/compiler/contextualTypingWithFixedTypeParameters1.ts(3,32): error TS
                          ~~~
 !!! error TS2339: Property 'foo' does not exist on type 'string'.
     var r9 = f10('', () => (a => a.foo), 1); // error
-             ~~~
-!!! error TS2453: The type argument for type parameter 'T' cannot be inferred from the usage. Consider specifying the type arguments explicitly.
-!!! error TS2453:   Type argument candidate 'string' is not a valid type argument because it is not a supertype of candidate 'T'.
                                    ~~~
-!!! error TS2339: Property 'foo' does not exist on type 'T'.
\ No newline at end of file
+!!! error TS2339: Property 'foo' does not exist on type 'string'.
+                                         ~
+!!! error TS2345: Argument of type 'number' is not assignable to parameter of type 'string'.
```

Comment:
I don't think we *need* to do it, but it certainly would be better. Either way, it's another orthogonal issue that isn't really related to the PR.